meteor/lib/api/pubsub.ts

```diff
@@ -59,7 +103,96 @@ export enum PubSub {
 	expectedPackagesForDevice = 'expectedPackagesForDevice',
 }
 
-export function meteorSubscribe(name: PubSub, ...args: any[]): Meteor.SubscriptionHandle {
+export interface PubSubTypes {
```

Comment:
Should these go into `shared-lib` instead?

Or perhaps the intention is for these to be internal in Core, if so perhaps that could be documented in the code, or even rename to something like `InternalPubSubTypes`? (Or, perhaps such a change would be included in the upcoming PublicAPI-discussion/task.)

Reply:
Ideally the ones which are relevant to the gateways should be defined in shared-lib, but that adds a lot of typings complications. 
The first problem I hit when looking at moving `timelineForDevice` was that it relies on TSR types, and we don't have that in shared-lib (yet?). Once I found that I gave up on it for now.
Perhaps this is another section of the shared-lib we should pair program? 

Im not sure about the naming for now, I think that will depend on how we decide to split it up between what is for the ui and what is for gateways. (should the gateway bits also be available for the ui?). Are there other valid use cases too that should have their own set?

Reply:
Other possible use cases could be future UI plug-ins, or a public API, or other specific Gateways (package manager, mos-gw, playout-gw, input/xkeys/streamdeck/vsm gateway)
Let's have a look at it together some time later.